legal-copy/terms-of-use.html

```diff
@@ -0,0 +1,101 @@
+
```

Comment:
Same comment as `privacy.html` - shouldn't be checked in since it's a build artifact